### PR TITLE
Fix local docker-compose

### PR DESCRIPTION
**Pull request by edgibbs**
A good bit of experimentation, but this appears to solve all the issues we saw today.  Anyway, I'll send out the updates the .env file independent of github.

### CODE REVIEW

**Comment by SashaKyz** [comment]
Can one of the admins verify this patch?

File: docker-compose.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@ services:
     ports:
       - "3000:3000"
       - "3035:3035"
+    environment:
+      SECRET_KEY_BASE: ${SECRET_KEY_BASE}
```
Comment (eckdanny-osi) [inline docker-compose.yml]: Consider instead:

```yml
 - SECRET_KEY_BASE
 - CASE_API_BASE_URL
 - PERRY_API_BASE_URL
```

 > "You can pass environment variables from your shell straight through to a service’s containers with the ‘environment’ key by not giving them a value"
> -[Docker Docs](https://docs.docker.com/compose/environment-variables/#passing-environment-variables-through-to-containers)

**Comment by eckdanny-osi** [comment]
Re the `dotenv-rails` gem: We should clarify as a group how we will manage ENV vars consistently (e.g.; in ansible, `docker[-compose]`, `[*].env` file(s), ...) 

I have a local refspec off of 3c88d1648a82353d0f58723c01a70c986d3757af (from Jan 7) that explores a [`dotenv`](https://github.com/bkeepers/dotenv)/[`figaro`](https://github.com/laserlemon/figaro) approach. A [currently open PR](https://github.com/ca-cwds/case-management/pull/187/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R73) _also_ appears to depend on `dotenv`

I think we should pick a means by which to manage env vars and merge a single-responsibility commit (e.g.; this pr!)

@edgibbs @RamuKammagani-osi @eckdany-osi

**Comment by amanbop** [comment]
@edgibbs @eckdanny-osi yes we need a consistent way to manage env variables.

**Comment by amanbop** [comment]
Tested changes. They are working.

**Comment by eckdanny-osi** [comment]
@amanbop @edgibbs I cross-linked the standing issue for ENV var stuff (#176) to related PRs. I'll add some detail to the issue to better capture the ask

**Pull request merged**